sdk/iot/Azure.Iot.Hub.Service/tests/prerequisites/setup.ps1

```diff
@@ -100,7 +101,8 @@ Write-Host("Writing user config file - $fileName`n")
 $appSecretJsonEscaped = ConvertTo-Json $appSecret
 $config = @"
 {
-    "IotHubHostName": "https://$($iotHubHostName)",
+    "IotHubHostName": "$iotHubHostName",
```

Comment:
Hostname shouldn't have "https" in it

Reply:
why not?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/URL/hostname

Reply:
I misunderstood this. I though you meant use http instead of https. Makes sense!